code/game/gamemodes/changeling/changeling.dm

```diff
@@ -289,7 +289,8 @@ var/list/slot2type = list("head" = /obj/item/clothing/head/changeling, "wear_mas
 	var/changelingID = "Changeling"
 	var/geneticdamage = 0
 	var/isabsorbing = 0
-	var/geneticpoints = 10
+	var/geneticpoints = 5 //Currency used for purchasing powers - gained from absorption
+	var/total_genetic_points = 5 //Used in respeccing to reset points to the proper amount

```

Comment:
This is used for respeccing. Whenever you absorb someone. that amount is upped but can't be modified like geneticpoints. Respeccing just sets geneticpoints to total_genetic_points.